runelite-client/src/main/java/net/runelite/client/plugins/poh/PohIcons.java

```diff
@@ -86,6 +86,8 @@
 	),
 	DIGSITEPENDANT("digsitependant",
 		DIGSITE_PENDANT, DIGSITE_PENDANT_33417, DIGSITE_PENDANT_33418, DIGSITE_PENDANT_33420
+	),
+	MYTHICALCAPE("mythicalcape", MYTHICAL_CAPE, MOUNTED_MYTHICAL_CAPE
```

Comment:
can you make this code style consistent with the rest of the file?

Reply:
Glory, Pools, Repair, etc are all one liners as well.

Reply:
Move the closing paren from the next line to the same line so this line reads:

```
	MYTHICALCAPE("mythicalcape", MYTHICAL_CAPE, MOUNTED_MYTHICAL_CAPE),
```